autotest/t007_test.py

```diff
@@ -172,8 +172,8 @@ def test_output_helper_shapefile_export():
     head = flopy.utils.HeadFile(os.path.join(ws, 'freyberg.hds'))
     cbc = flopy.utils.CellBudgetFile(os.path.join(ws, "freyberg.cbc"))
     flopy.export.utils.output_helper(os.path.join('temp', 'test.shp'), ml,
-                                     {'HDS': head, 'cbc': cbc},
-                                     mflay=1, kper=10)
+                                      {'HDS': head, 'cbc': cbc},
```

Comment:
There are changes scattered throughout this PR where a continued line is indented further by one space.  I think we want to align with opening delimiter (https://www.python.org/dev/peps/pep-0008/#indentation), so I don't think we want these indentation changes.

Reply:
This was an unwanted result of commenting and uncommenting bocks in Spyder editor; do not know why Spyder has this weird indentation habit... anyways, I corrected these.